content/docs/fairing/gcp-local-notebook.md

```diff
@@ -0,0 +1,230 @@
+
+## Clone the Kubeflow Fairing repository
+
+Clone the Kubeflow Fairing repository to download the files used in this example.
+
+```bash
+git clone https://github.com/kubeflow/fairing 
+cd fairing
+```
+
+## Set up Python, Jupyter Notebook, and Kubeflow Fairing
+
+1.  You need **Python 3.6** or later to use Kubeflow Fairing. To check if
+    you have Python 3.6 or later installed, run the following command:
+
+    ```bash
+    python3 -V
+    ```
+
+    The response should be something like this:
+
+    ```
+    Python 3.6.5
+    ```
+
+    If you do not have Python 3.6 or later, you can [download
+    Python](https://www.python.org/downloads/) from the Python Software
+    Foundation.
+
+1.  Use virtualenv to create a virtual environment to install Kubeflow
+    Fairing in. To check if you have virtualenv installed, run the
+    following command: 
+
+    ```bash
+    which virtualenv
+    ```
+
+    The response should be something like this.
+
+    ```bash
+    /usr/bin/virtualenv
+    ```
+
+    If you do not have virtualenv, use pip3 to install virtualenv.
+
+    ```bash
+    pip3 install --upgrade virtualenv
+    ```
+
+    Create a new virtual environment, and activate it.
+
+    ```bash
+    virtualenv venv --python=python3
+    source venv/bin/activate
+    ```
+
+1.  Install Jupyter Notebook.
+
+    ```bash
+    pip3 install --upgrade jupyter
+    ```
+
+1.  Install Kubeflow Fairing from the cloned repository.
+
+    ```bash
+    pip3 install --upgrade .
+    ```
+
+1.  Install the Python dependencies for the XGBoost demo notebook.
+
+    ```bash
+    pip3 install -r examples/prediction/requirements.txt
+    ```
+
+## Install and configure the Google Cloud SDK
+
+In order to use Kubeflow Fairing to train or deploy to Kubeflow on GKE,
+or Cloud Machine Learning Engine, you must configure
+your development environment with access to GCP. 
+
+1.  If you do not have the Cloud SDK installed, [install the
+    Cloud SDK][gcloud-install].
+
+1.  Use `gcloud` to set a default project.
+
+    ```bash
+    export PROJECT_ID=<your-project-id>
+    gcloud config set project $PROJECT_ID
+    ```
+
+1.  Kubeflow Fairing needs a service account to make API calls to GCP. The
+    recommended way to provide Fairing with access to this
+    service account is to set the `GOOGLE_APPLICATION_CREDENTIALS` environmental
+    variable. To check for the `GOOGLE_APPLICATION_CREDENTIALS` environmental
+    variable, run the following command:
+
+    ```bash
+    ls "$GOOGLE_APPLICATION_CREDENTIALS"
+    ```
+
+    The response should be something like this:
+
+    ```bash
+    /.../.../key.json
+    ```
+
+    If you do not have a service account, then create one and grant it
+    access to the required roles.
+
+    ```bash
+    export SA_NAME=<your-sa-name>
+    gcloud iam service-accounts create $SA_NAME
+    gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:$SA_NAME@$PROJECT_ID.iam.gserviceaccount.com  --role 'roles/editor'
```

Comment:
The rendering of this code block is not ideal. The long command is putting a horizontal scroll that hides the last command. Probably split the long line using "\"?

Reply:
Did you mean to split the long line with a \ or ""? I split the line with a \, let me know if you would like it split differently.